experimental/dispatch_profiler/launchers.py

```diff
@@ -0,0 +1,194 @@
+      )
+
+      compile_log_filename = f"{self.operation_path}/iree_compile_logs.mlir"
+      with open(compile_log_filename, "w") as fp:
+        subprocess.run(cmd, stderr=fp)
+
+    elif self.args.verbose:
+      print("Skipping compilation of operation: " + vmfb_file +
+            " since it already exists.")
+
+  def verify(self, configuration):
+    """Verifies the operation with a given configuration."""
+    # First compile the operation to a vmfb file.
+    self.iree_compile(CompilationMode.Verify)
+
+    # Verify using random data distribution.
+    reference_run = self.reference_impl_map[self.operation.operation_kind](\
+        self.operation, self.op_reference_cache_path, Distribution.Random, Distribution.Random)
+
+    if not reference_run.is_cached():
+      reference_run()
+
+    # Commandline `iree-run-module` for verification.
+    cmd = [
+        self.iree_run_module_path, f'--module={self.vmfb_verify_filepath}',
+        f'--device={self.args.device}'
+    ]
+
+    # Operation-specific verification command-line.
+    cmd.append(f'--function={self.operation.name()}_{configuration.name()}')
+    for input_file_path in reference_run.get_input_filepaths():
+      cmd.append(f'--input=@{input_file_path}')
+    for output_file_path in reference_run.get_output_filepaths():
+      cmd.append(f'--expected_output=@{output_file_path}')
+
+    # Print the command if verbose.
+    if self.args.verbose:
+      print(">> Verification command: " + ' '.join(cmd))
+
+    # Launch verification.
+    cmd_output = subprocess.check_output(cmd, text=True)
+
+    # Parse the verification output.
+    m = re.search(r"\[(?P<verification_result>[a-zA-Z]+)\]", cmd_output)
+    if m is None:
+      raise Exception(
+          "Failed to parse verification output by iree-run-module: " +
+          cmd_output)
+    verification_result = m.group('verification_result')
+
+    if self.args.verbose or verification_result != "SUCCESS":
+      print(cmd_output)
+
+    return verification_result
+
+  def profile(self, configuration):
+    """Profiles the operation with a given configuration."""
+    # First compile the operation to a vmfb file.
+    self.iree_compile(CompilationMode.Profile)
+
+    # Commandline `iree-benchmark-module` for profiling.
+    cmd = [
+        self.iree_benchmark_module_path,
+        f'--module={self.vmfb_benchmark_filepath}',
+        f'--device={self.args.device}'
+    ]
+
+    # Profiling specific flags.
+    cmd += [f'--benchmark_repetitions={self.args.benchmark_repetitions}']
+    cmd += [f'--batch_size={self.batch_size}']
+
+    # Operation-specific profiling command-line.
+    cmd += [f'--function={self.operation.name()}_{configuration.name()}']
+    cmd += [f'--input={self.operation.lhs_npy_shape()}']
+    cmd += [f'--input={self.operation.rhs_npy_shape()}']
+
+    # Print the command if verbose.
+    if self.args.verbose:
+      print(">> Profiling command: " + ' '.join(cmd))
```

Comment:
Same here. It might be useful to dump the verify/benchmark flags into flagfiles for reproducing manually.